README.md

```diff
@@ -49,6 +49,32 @@ cd .\azcopy_windows_amd64_10.0.8
 .\azcopy.exe
 ```
 
+### Installation for repeatable scripts
+
+The version of `azcopy` available at `https://aka.ms/downloadazcopy-v10-linux` will be updated over time. If you plan on using `azcopy` as part of a script it's best to pin to a known version so future updates don't affect your scripts. 
+
+To pin to a version of `azcopy`
+
+1. Run `curl -v https://aka.ms/downloadazcopy-v10-linux` and review the output to locate the current version URL. Here is an example output from `curl` showing the `301 Moved Permanently` response.
```

Comment:
Windows equivalent is this PowerShell command:
` (curl https://aka.ms/downloadazcopy-v10-windows -MaximumRedirection 0 -ErrorAction silentlycontinue).RawContent`

Reply:
I've added this one in now - good call